PWA/src/components/Timesheet/ConfirmSubmission.vue

```diff
@@ -157,16 +150,139 @@
         //Data to be submitted
         submitData: null,
 
+        // All the errors of this form
+        errors: [],
+        isValid: this.valid,
+
         //URL for the AppServer
         url: process.env.VUE_APP_SERVER_URL.concat("Submit"),
       };
     },
 
+    watch: {
+      valid(newVal) {
+        this.isValid = newVal;
+      },
+    },
+
     methods: {
+      // Compute the sum of all serviceDeliveredOn totalHours with the totalHours field
+      sumTableHours() {
+        var sumHours = 0;
+        var sumMinutes = 0;
+
+        // For each row in the array of entries...
+        this.formFields["serviceDeliveredOn"]["value"].forEach((entry) => {
+          // Check that the totalHours field is valid
+          if (entry["errors"]["totalHours"].length == 0) {
```

Comment:
this wasn't working as expected in the table, it would say a row is invalid if even if total hours was correct.